kindergarten-garden/kindergarten-garden_spec.lua

```diff
@@ -0,0 +1,52 @@
+local Garden = require('kindergarten-garden')
+
+describe('kindergarten garden', function()
+
+  it('should be able to parse a garden with only one student', function()
+    local garden = Garden('RC\nGG')
+    assert.same({ 'radishes', 'clover', 'grass', 'grass' }, garden.alice)
```

Comment:
I chose to index the object with the student name to reward the use of metatables